src/jobs/enrich-mustache-from-path-filter.yml

```diff
@@ -0,0 +1,79 @@
+description: >-
+  meant to be a drop in replacement for Circle's path-filtering/filter
+  BUT: meant to be used in a situation where EITHER you
```

Comment:
Nit: Would prolly make this a bullet point list instead of the caps with `BUT`/`EITHER`